resolwe_bio/utils/test.py

```diff
@@ -97,6 +97,16 @@ def prepare_expression(self, f_rc='exp_1_rc.tab.gz',
         inputs = {'rc': f_rc, 'exp': f_exp, 'exp_type': f_type, 'exp_name': name, 'source': source}
         return self.run_process('upload-expression', inputs)
 
+    def assertAlmostEqualList(self, actual, expected):
```

Comment:
I think this would be also useful in Resolwe. What do you think?

Reply:
Moved to genialis/resolwe#350.

Reply:
Perfect, thanks!